account_reconcile_model_strict_match_amount/tests/test_account_reconcile_model_strict_match_amount.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.addons.account.tests.test_reconciliation_matching_rules import TestReconciliationMatchingRules
+
+
+class TestAccountReconcileModelStrictMatchAmount(TestReconciliationMatchingRules):
+
+    def setUp(self):
+        super().setUp()
+        self.partner_3 = self.env['res.partner'].create({'name': 'partner_3'})
+        self.partner_4 = self.env['res.partner'].create({'name': 'partner_4'})
+        self.invoice_line_5 = self._create_invoice_line(
+            150, self.partner_3, 'out_invoice'
+        )
+        self.invoice_line_5.ref = 'ABC001XYZ'
+        self.invoice_line_6 = self._create_invoice_line(
+            300, self.partner_4, 'out_invoice'
+        )
+        self.invoice_line_5.name = 'ABC002XYZ'
```

Comment:
```suggestion
        self.invoice_line_6.name = 'ABC002XYZ'
```
:warning: test pass but this is wrong